js/ui/validator.js

```diff
@@ -205,10 +205,18 @@ var Validator = DOMComponent.inherit({
         }
     },
 
-    _resetValidationState: function() {
-        each(this.option("validationRules"), function(_, rule) {
-            delete rule.isValid;
-        });
+    _getValidationRules: function() {
+        if(!this._validationRules) {
+            this._validationRules = map(this.option("validationRules"), (function(rule) {
```

Comment:
Please inspect [this](https://github.com/DevExpress/DevExtreme/blob/18_1/js/integration/knockout/validation.js#L18) file. I saw a similar code for KO.